### PR TITLE
feat: add `--line-length` option to `melos format` command

### PR DESCRIPTION
**Pull request by utamori**
## Description

This pull request adds a `--line-length` option to the `melos format` command, 
allowing users to specify the desired line length when formatting Dart code across packages in a Melos workspace.

## Type of Change

- [x] ✨ `feat` -- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: packages/melos/lib/src/command_runner/format.dart [hunk 1]
```diff
@@ -19,6 +19,11 @@ class FormatCommand extends MelosCommand {
           '[write]              Overwrite formatted files on disk.\n',
       abbr: 'o',
     );
+    argParser.addOption(
+      'line-length',
+      defaultsTo: '80',
```
Comment (spydon) [inline packages/melos/lib/src/command_runner/format.dart:24]: I think it's better to have this nullable and just not add the flag in the cases when this isn't passed in.

Comment (utamori) [reply packages/melos/lib/src/command_runner/format.dart]: Thank you for your feedback

File: packages/melos/lib/src/command_runner/format.dart [hunk 2]
```diff
@@ -32,6 +36,7 @@ class FormatCommand extends MelosCommand {
     final setExitIfChanged = argResults?['set-exit-if-changed'] as bool;
     final output = argResults?['output'] as String?;
     final concurrency = int.parse(argResults!['concurrency'] as String);
+    final lineLength = int.parse(argResults!['line-length'] as String);
```
Comment (spydon) [inline packages/melos/lib/src/command_runner/format.dart:39]: This needs to accept null too

**Review by spydon — DISMISSED** [review]

File: packages/melos/lib/src/command_runner/format.dart (hunk 2 above)
Comment (utamori) [reply packages/melos/lib/src/command_runner/format.dart]: I missed it. thanks.

**Review by spydon — APPROVED** [review]

**Pull request merged**